aspnetcore/signalr/get-started-signalr-core.md

```diff
@@ -0,0 +1,107 @@
+---
+uid: signalr/get-started-signalr-core
+title: Get Started with SignalR on ASP.NET Core
+author: rachelappel
+ms.author: rachelap
+description: In this tutorial, you will create a SignalR on ASP.NET Core application.
+manager: wpickett
+ms.date: 02/23/2018
+ms.topic: tutorial
+ms.technology: dotnet-signalr
+ms.prod: aspnet-core
+ms.custom: mvc
+---
+# Tutorial: Get Started with SignalR for ASP.NET Core
+
+By [Rachel Appel](https://twitter.com/rachelappel)
+
+This tutorial teaches the basics of building a real-time app using SignalR for ASP.NET Core. 
+
+This tutorial demonstrates the following SignalR development tasks:
+
+> [!div class="checklist"]
+> * Add the SignalR library to an ASP.NET web application.
+> * Create a SignalR hub to push content to clients.
+> * Modify the `Startup` class to configure the application.
+> * Use the SignalR JavaScript library to send messages and display updates from the hub.
+
+# Prerequisites
+
+Install the following:
+
+* [.NET Core 2.0.0 SDK](https://www.microsoft.com/net/core) or later.
+* [Visual Studio 2017](https://www.visualstudio.com/downloads/) version 15.6 or later with the ASP.NET and web development workload.
```

Comment:
We should get rid of VS or use tabs. You shouldn't need VS to do this.

Reply:
If you remove VS from the list, then you'll need to add npm.

Reply:
We should keep this as cross-platform as we can. I don't mind if the tutorial focuses on the VS scenario but perhaps if we mention how to do things on non-Windows (`dotnet new razor ...` instead of `New Project...`, etc.).